NRasterizer.Tests/FakeGlyphRasterizer.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace NRasterizer.Tests
+{
+    public class FakeGlyphRasterizer : IGlyphRasterizer
+    {
+        public int BeginReadCallCount { get; private set; }
+        public int CloseFigureCallCount { get; private set; }
+        public int ContourCount { get; private set; }
+        public int EndReadCallCount { get; private set; }
+        public int FlushCallCount { get; private set; }
+        public List<Point<double>> Points { get; set; } = new List<Point<double>>();
+        public int Resolution { get; set; } =  72;
+
+        public void BeginRead(int countourCount)
+        {
+            this.BeginReadCallCount++;
+            this.ContourCount = countourCount;
```

Comment:
It's fine to merge also, it's probably faster for me to go through smaller things like this and trailing newlines etc in one go afterwards.